client/src/components/Footer/modal.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./modal.css";
+import ShowName from "./showName";
+
+function Modal() {
+  const [userData, setUserData] = useState([]);
+
+  useEffect(() => {
+    getGitHubUserWithFetch();
+  }, []);
+
+  const getGitHubUserWithFetch = async () => {
+    const response = await fetch(
```

Comment:
How about using axios and promises to handle the fetch?

Reply:
I don't know.
For now, it's good.